packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -154,7 +154,17 @@
         format("woff");
 }
 
+/* Axis-related styling */
+
 .MafsView .axis-ticks text {
     font-size: 14px;
     font-family: "Mafs-MJXTEX";
 }
+
+/* This is a gross method of referencing the axis lines within the graph,
+        but there are no other identifying traits that we can use to target these lines
+        since they are controlled by Mafs.
+ */
```

Comment:
Indeed.  So much so that I'd be inclined to see if we can upstream the line thickness as an option.  I'd like to avoid hacks like this while we're in active development.  Steven is very open to us proposing changes.

Reply:
I'll draw up a PR for Mafs.

Reply:
Mafs has been updated (v0.18.6), and the latest commit to Perseus includes that update.